tests/unit/scripts/helpers/classnames.test.js

```diff
@@ -0,0 +1,43 @@
+import { classnames } from "../../../../scripts/helpers/classnames";
+import { selector } from "../../../../scripts/helpers/selector";
+import { getComponentManifest } from "../../../helpers/blocks";
+
+it('tests classnames returns class string from manifest values', () => {
+	const accordionManifest = getComponentManifest('accordion');
+
+	const accordionClassnames = classnames([
+		selector(accordionManifest.componentClass, accordionManifest.componentClass),
+		selector(accordionManifest.componentJsClass, accordionManifest.componentJsClass),
+	]);
+
+	expect(accordionClassnames).toBe(`${accordionManifest.componentClass} ${accordionManifest.componentJsClass}`);
+});
+
+it('tests classnames returns empty class string when selector function returns empty string', () => {
+	const accordionManifest = getComponentManifest('accordion');
+
+	const accordionClassnames = classnames([
+		'',
+		' ',
+		selector(!accordionManifest.componentClass, accordionManifest.componentClass),
+		selector(!accordionManifest.componentJsClass, accordionManifest.componentJsClass),
+	]);
+
+	expect(accordionClassnames).toBe('');
+});
+
+it('tests classnames returns only valid classes', () => {
```

Comment:
Can you add a test where you have a ternary as one of the arguments in the classnames array? Just so that we cover all the cases. 

Also, can you think of any 'unhappy path' tests - the test where you give it invalid input and expect a correct error?

Thanks!

Reply:
Just gonna leave this here :eyes: https://stackoverflow.com/questions/448981/which-characters-are-valid-in-css-class-names-selectors

Reply:
I've added additional case when ternary operator is used as an item in class names array.

As for the invalid CSS class names, I think this helper shouldn't worry about that, since this is not its domain.
The purpose of this helper is to create a string of class names out of array of potential class names. Invalid classnames have more to do with CSS for styling and JS for adding interactions.  

Also how would this be tested? Running a `document.querySelector()` method and evaluating the results seems kind of redundant and a large overhead in this case.